src/screens/error/ErrorScreen.tsx

```diff
@@ -75,13 +83,17 @@ const handleMessage = async (
   )
 
 const makeSource = (route: ErrorScreenProps['route']): Source => {
```

Comment:
maybe it's time to decouple this function from the view file if we keep this pattern. It's starting to grow big